torch/csrc/distributed/rpc/request_callback_no_python.cpp

```diff
@@ -171,23 +172,62 @@ void RequestCallbackNoPython::processPythonRemoteCall(
   C10_THROW_ERROR(Error, "Python call not supported!");
 }
 
-void RequestCallbackNoPython::processRRefBackward(
-      RpcCommandBase& rpc,
-      const int64_t messageId,
-      const std::shared_ptr<FutureMessage>& responseFuture) const {
-  C10_THROW_ERROR(Error, "Python call not supported!");
-}
-
 void RequestCallbackNoPython::processScriptRemoteCall(
     ScriptRemoteCall& scriptRemoteCall,
     const std::function<void(void)>& postProcessing,
     std::vector<at::IValue>& stack,
     const c10::intrusive_ptr<OwnerRRef>& ownerRRef) const {
   TORCH_CHECK(
-      scriptRemoteCall.hasOp(), "ScriptRemoteCall needs to have an op!");
+    scriptRemoteCall.hasOp(), "ScriptRemoteCall needs to have an op!");
   processScriptRemoteCallOp(scriptRemoteCall, postProcessing, stack, ownerRRef);
 }
 
+void RequestCallbackNoPython::processBaseScriptRemoteCall(
```

Comment:
Curious, what's the reason for splitting `ScriptRemoteCall` into `Base*` and `*Op`?

Reply:
I needed to add the Base* to keep the processing logic the consistent since ScriptRemoteCall is defined in request_callback_impl.cpp. It would be helpful to understand what were the original goals and distinctions between request_callback_impl.cpp and request_callback_no_python.cpp. I'm still a bit fuzzy on that